test/e2e/utils.go

```diff
@@ -80,6 +84,16 @@ func writePodsLogs(namespace string, writer io.Writer) error {
 	return nil
 }
 
+func interfacesName(interfaces []map[string]interface{}) []string {
+	var names []string
+	for _, iface := range interfaces {
+		name, hasName := iface["name"]
+		Expect(hasName).To(BeTrue())
```

Comment:
If we fail on this, all we will see is that False is not True in this function. Please add here (and all Expects where it makes sense) message describing why we failed and were we checking

Reply:
What would be nice is to validate this towards (nmstate schmea)[https://github.com/nmstate/nmstate/blob/master/libnmstate/schemas/operational-state.yaml]

Reply:
I'd suggest open an issue for that. Sounds like a nice little self-contained task.

Reply:
Done, but only for test, for the operator this have to be a feature after we generate the config from policies.

https://github.com/nmstate/kubernetes-nmstate/issues/76